corehq/apps/data_dictionary/views.py

```diff
@@ -452,7 +448,7 @@ def _process_bulk_upload(bulk_file, domain):
                         continue
                     if row_len < 3:
                         # if missing value or description, fill in "blank"
-                        row += [Cell(value='') for _ in range(3 - row_len)]
+                        row += [Cell(value='') for __ in range(3 - row_len)]
```

Comment:
:star: 
This is just a linting thing, right?  I believe python 3 fixed this issue by limiting the scope of list comprehension iterables to inside the list comprehension:
```python
In [1]: _ = 'foo'

In [2]: ['hello' for _ in range(5)]
Out[2]: ['hello', 'hello', 'hello', 'hello', 'hello']

In [3]: _
Out[3]: 'foo'
```

Reply:
Ah! This is good to know! PyCharm was shouting red lines at me, which is what motivated this PR.

I've just checked, and I have not configured PyCharm for Python 2.7 compatibility :thinking: so maybe this is a bug in its error checking /  scoping.